libgraph/include/katana/GraphTopology.h

```diff
@@ -43,6 +45,59 @@ struct KATANA_EXPORT GraphTopologyTypes {
   using EdgeDestVec = NUMAArray<Node>;
   using PropIndexVec = NUMAArray<PropertyIndex>;
   using EntityTypeIDVec = NUMAArray<EntityTypeID>;
+  */
+
+  struct Node: public OpaqueIDLinear<Node, uint32_t> {
+    using OpaqueIDLinear::OpaqueIDLinear;
+  };
+
+  struct Edge: public OpaqueIDLinear<Edge, uint64_t> {
+    using OpaqueIDLinear::OpaqueIDLinear;
+  };
+
+  struct OutEdgeHandle: public OpaqueIDLinear<OutEdgeHandle, uint64_t> {
+    using OpaqueIDLinear::OpaqueIDLinear;
+  };
+
+  struct InEdgeHandle: public OpaqueIDLinear<InEdgeHandle, uint64_t> {
+    using OpaqueIDLinear::OpaqueIDLinear;
+  };
+
+  struct InOutEdgeHandle: public OpaqueIDLinear<InOutEdgeHandle, uint64_t> {
+    using OpaqueIDLinear::OpaqueIDLinear;
+  };
+
```

Comment:
What about the `NodeHandle`? It's for situations where we sort nodes.

Reply:
Good point. We should add it.